### ISSUE TITLE
[Accessibility] [ContextualMenu] Validation fails for button name

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1042]
The contextual menu component fails accessibility validation for button name.

Details:
https://dequeuniversity.com/rules/axe/2.1/button-name

Error displayed:
![image](https://cloud.githubusercontent.com/assets/11937034/23124400/d0c54872-f792-11e6-8cc7-b9dec459c736.png)

**Comment by mdahamiwal** [comment]
@prmittal, I am not able to repro the failure with chrome accessibility audit run. Also, the code snippet referred above doesn't have a button role which is mentioned at https://dequeuniversity.com/rules/axe/2.1/button-name . Can you provide more details on how to repro this and the side effects of this validation failure?

**Comment by dzearing** [comment]
Closing due to no traction. Please feel free to reopen with more information.
